specification/hybridcompute/resource-manager/Microsoft.HybridCompute/stable/2020-08-02/HybridCompute.json

```diff
@@ -231,7 +231,8 @@
             "type": "string",
             "description": "The expand expression to apply on the operation.",
             "enum": [
-              "instanceView"
+              "instanceView",
+              "userData"
```

Comment:
Does userData need to be added here? This is under Microsoft.HybridCompute, which is a different RP I assume.

Reply:
Yes, in order to add a new enum value to InstanceViewTypes (or any other existing enum), the enum value has to be added to all instances of InstanceViewTypes.
Otherwise, the PR check and AutoRestGeneration will fail with this error

“Swagger document contains two or more x-ms-enum extensions with the same name 'InstanceViewTypes' and different values: instanceView,userData vs. instanceView”

Reply:
This is different RP, we should not be making changes to the swagger file of a different RP, something is incorrect if we are expected to make changes to types of a different RP.